src/reducers/computeRenderedRows.js

```diff
@@ -55,7 +55,8 @@ export default function computeRenderedRows(state, scrollAnchor) {
     newState.firstRowOffset = 0;
   }
 
-  computeRenderedRowOffsets(newState, rowRange);
+  computeRenderedRowOffsets(newState, rowRange, state.scrolling);
```

Comment:
cleanup for:
* https://github.com/schrodinger/fixed-data-table-2/pull/415#discussion_r269329250
* https://github.com/schrodinger/fixed-data-table-2/pull/415#discussion_r269329945